# Generated by Django 4.2 on 2024-10-07 05:10

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('health', '0019_document_ref_num'),
    ]

    operations = [
        migrations.CreateModel(
            name='CeleryTaskmeta',
            fields=[
                ('id', models.IntegerField(primary_key=True, serialize=False)),
                ('task_id', models.CharField(blank=True, max_length=155, null=True, unique=True)),
                ('status', models.CharField(blank=True, max_length=50, null=True)),
                ('result', models.BinaryField(blank=True, null=True)),
                ('date_done', models.DateTimeField(blank=True, null=True)),
                ('traceback', models.TextField(blank=True, null=True)),
                ('name', models.CharField(blank=True, max_length=155, null=True)),
                ('args', models.BinaryField(blank=True, null=True)),
                ('kwargs', models.BinaryField(blank=True, null=True)),
                ('worker', models.CharField(blank=True, max_length=155, null=True)),
                ('retries', models.IntegerField(blank=True, null=True)),
                ('queue', models.CharField(blank=True, max_length=155, null=True)),
            ],
            options={
                'db_table': 'celery_taskmeta',
                'managed': False,
            },
        ),
        migrations.AddField(
            model_name='datasynclog',
            name='status',
            field=models.CharField(choices=[('processing', 'processing'), ('completed', 'completed'), ('failed', 'failed')], default='processing', max_length=100),
        ),
        migrations.AddField(
            model_name='userservicetoken',
            name='is_initial_sync',
            field=models.BooleanField(default=False),
        ),
        migrations.AddField(
            model_name='userservicetoken',
            name='last_sync_date',
            field=models.DateTimeField(blank=True, null=True),
        ),
    ]
